.github/dependabot.yml

```diff
@@ -1,7 +1,6 @@
 version: 2
 updates:
   - package-ecosystem: npm
```

Comment:
Not sure how relevant this is, but apparently "pnpm" is it's own package ecosystem 
-> https://docs.github.com/en/code-security/dependabot/ecosystems-supported-by-dependabot/supported-ecosystems-and-repositories

Reply:
yes, the value in the yaml file is `npm` for that.